boards/atxmega-a1u-xpro/Makefile.features

```diff
@@ -1,3 +1,9 @@
 CPU_MODEL = atxmega128a1u
 
+FEATURES_PROVIDED += periph_gpio periph_gpio_irq
+FEATURES_PROVIDED += periph_i2c
+FEATURES_PROVIDED += periph_spi
+FEATURES_PROVIDED += periph_timer periph_timer_periodic
```

Comment:
`periph_timer_periodic` should stay a CPU feature as it's more a hint of that the implementation support that feature - it does not require any board specific configuration.